pallets/elections/src/impls.rs

```diff
@@ -21,6 +22,7 @@ pub const LENIENT_THRESHOLD: Perquintill = Perquintill::from_percent(90);
 ///
 /// 1. Block `B` initialized
 /// 2. `end_session(S)` is called
+/// -  Based on block count we might mark the session for a given validator as undeperformed
```

Comment:
could you also add a point when the kick might happen

Reply:
will do

Reply:
done